examples/point_cloud/CMakeLists.txt

```diff
@@ -13,13 +13,10 @@ endif(MSVC)
 
 
 
-set(SRC_DIR point_cloud)
-set(DOCOPT_SRC ./third_party/doc_opt/)
-set(TOFI_CCB ./ccb/)
-set(CMAKE_CURR_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR})
-set (CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_CURRENT_BINARY_DIR}/bin/)
-set (CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_CURRENT_BINARY_DIR}/bin/)
-set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin/)
+set(SRC_DIR ./)
+set(DOCOPT_SRC ../data_collect/third_party/doc_opt/)
```

Comment:
I would use  ${CMAKE_SOURCE_DIR} as a starting point to get to this paths. This way we avoid any problems if somebody creates the build folder outside of the sdk.